### PR TITLE
FCM Options Support in WebpushConfig API

### PR DESCRIPTION
**Pull request by hiranya911**
Adding the `fcm_options` parameter to the `WebpushConfig` type. Change ported from https://github.com/firebase/firebase-admin-node/pull/373

Resolves #203

### CODE REVIEW

**Comment by hiranya911** [comment]
Resolves #202

File: firebase_admin/_messaging_utils.py [hunk 1]
```diff
@@ -232,6 +234,18 @@ def __init__(self, title=None, body=None, icon=None, actions=None, badge=None, d
         self.custom_data = custom_data
 
 
+class WebpushFcmOptions(object):
+    """Options for features provided by the FCM SDK for Web.
+
+    Args:
+        link: The link to open when the user clicks on the notification. Must be a HTTPS URL
```
Comment (bklimt) [inline firebase_admin/_messaging_utils.py]: s/a HTTPS/an HTTPS/

File: firebase_admin/_messaging_utils.py [hunk 2]
```diff
@@ -588,6 +603,20 @@ def encode_webpush_notification_actions(cls, actions):
...
+    def encode_webpush_fcm_options(cls, options):
+        """Encodes a WebpushFcmOptions instance into JSON."""
+        if options is None:
+            return None
+        result = {
+            'link': _Validators.check_string('WebpushConfig.fcm_options.link', options.link),
+        }
+        result = cls.remove_null_values(result)
+        link = result.get('link')
+        if link is not None and not link.startswith('https://'):
```
Comment (bklimt) [inline firebase_admin/_messaging_utils.py]: Should we allow alternate capitalization on the protocol? I mean, "HTTPS://" is probably okay, right?

Comment (hiranya911) [reply firebase_admin/_messaging_utils.py]: Yes, but we have always checked only for the lower-cased prefix in this codebase (there are similar URL checks in auth, db etc). So I'll keep this consistent with that. Also developers haven't  requested this, so it should be ok.

File: firebase_admin/_messaging_utils.py (hunk 1 above)
Comment (hiranya911) [reply firebase_admin/_messaging_utils.py]: Done

**Pull request merged**